src/plugins/intel_cpu/src/emitters/snippets/brgemm_generic.cpp

```diff
@@ -146,25 +200,39 @@ std::tuple<int64_t, int64_t, int64_t, float> BrgemmKernelExecutorHelper::get_run
     auto K = *in0_subtensor.rbegin();
     auto N = *in1_subtensor.rbegin();
 
-    size_t loop_idx = 0;
     const auto& loop_ids = expr->get_loop_ids();
     const auto& loop_manager = linear_ir->get_loop_manager();
-    auto get_loop_info = [&]() {
-        assert(loop_idx < loop_ids.size() && "Loop is missed");
-        return loop_manager->get_loop_info<ov::snippets::lowered::ExpandedLoopInfo>(loop_ids[loop_idx++]);
-    };
+    const auto [m_loop_idx, n_loop_idx, k_loop_idx] = [&]() {
```

Comment:
Question: does it make sense to extract it to utils of some sort? Is it potentially reusable?